### PR TITLE
Fix/hierarchy enhancement

### PR DESCRIPTION
**Pull request by PROgram52bc**

### CODE REVIEW

File: ui/src/components/groups/treeview/GroupTreeViewHierarchy.vue [hunk 1]
```diff
@@ -185,9 +197,9 @@ export default {
           if (err instanceof HierarchyCycleError) {
             this.cycleError.show = true;
             this.cycleError.node = err.node;
-            this.cycleError.path = err.path.reverse();
+            this.cycleError.path = err.path;
             eventBus.$emit("error", {
-              content: err.message,
+              content: this.$t("groups.treeview.unexpected-cycle"),
```
Comment (PROgram52bc) [inline ui/src/components/groups/treeview/GroupTreeViewHierarchy.vue]: remove this.$t

File: api/src/groups/create_group_data.py [hunk 2]
```diff
@@ -274,60 +280,89 @@ def create_hierarchical_groups_and_participants(sqla):
...
+    for i in range(1, max_group_id + 1):
         group_name = f"Group{i}"
         group = Group(**group_schema.load(
             group_object_factory(group_type_id, name=group_name, active=True)))
         sqla.add(group)
         groups.append(group)
         print(group)
     sqla.commit()
+    print(people)
+    print(groups)
```
Comment (PROgram52bc) [inline api/src/groups/create_group_data.py]: remove print statement

File: ui/src/components/groups/GroupMeetingMember.vue [hunk 3]
```diff
@@ -133,14 +133,14 @@ export default {
           )
           .then(() => {
             eventBus.$emit("message", {
-              content: this.$t("groups.messages.participant-added"),
+              content: "groups.messages.participant-added",
```
Comment (nurkkala) [inline ui/src/components/groups/GroupMeetingMember.vue:136]: Nice -- I like this approach.

File: ui/src/components/groups/GroupTable.vue [hunk 4]
```diff
@@ -95,7 +95,9 @@
       must-sort
       :item-class="itemClass"
       class="elevation-1"
-      :footer-props='{itemsPerPageText: $t("$vuetify.dataTable.rowsPerPageText")}'
+      :footer-props="{
```
Comment (nurkkala) [inline ui/src/components/groups/GroupTable.vue]: Do we still need `:footer-props` with Qiang's I18N update?

**Review by nurkkala — COMMENTED** [review]
Glad to see you updating the tests along with the other changes.

File: ui/src/components/groups/GroupTable.vue (hunk 4 above)
Comment (PROgram52bc) [reply ui/src/components/groups/GroupTable.vue]: I will remove them now!

**Review by nurkkala — APPROVED** [review]
Much cleaner.  Nice.

**Pull request merged**